jni/dc_wrapper.c

```diff
@@ -307,6 +313,21 @@ JNIEXPORT jlong Java_com_b44t_messenger_DcContext_createContextCPtr(JNIEnv *env,
 }
 
 
+JNIEXPORT jint Java_com_b44t_messenger_DcContext_open(JNIEnv *env, jobject obj, jstring passphrase)
+{
+    CHAR_REF(passphrase);
+    jint ret = dc_context_open(get_dc_context(env, obj), passphrasePtr);
+    CHAR_UNREF(passphrase);
+    return ret;
+} // TODO should maybe return jboolean
```

Comment:
Should this return `jboolean`? (@r10s or others)

Reply:
on DC iOS I return a boolean: https://github.com/deltachat/deltachat-ios/pull/1438/files#diff-a5590eddfe26770d5692a338e715ab8f047049b4b63c9ea802f6de4d43b097c4R127

For now I'm only logging if opening fails though

Reply:
i would also return a `jboolean`, that makes the java part easier to read, so `return ret != 0` (ios uses `ret == 1` but i would keep the style used elsewhere here)